storio-test-common/src/test/java/com/pushtorefresh/storio/test/StorIOAssertTest.java

```diff
@@ -0,0 +1,39 @@
+package com.pushtorefresh.storio.test;
+
+import org.junit.Test;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import rx.Observable;
+import rx.functions.Action1;
+
+import static junit.framework.Assert.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class StorIOAssertTest {
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void assertThatObservableEmitsOncePositive() {
+        final Action1<String> testAction = mock(Action1.class);
+        final String testString = "Test string";
+        StorIOAssert.assertThatObservableEmitsOnce(Observable.just(testString), testAction);
```

Comment:
In that case `assertThatObservableEmitsOnce` is subject of a test, I am testing behavior of `assertThatObservableEmitsOnce`, so it will be better to call it as class method.

In other cases, where we just use `assertThatObservableEmitsOnce()` writing class name is not very cool :)

Take a look at PreparedPutObjectsTest.java line 39 :)